cmd/admin-info.go

```diff
@@ -165,8 +189,8 @@ func mainAdminInfo(ctx *cli.Context) error {
 	client, err := newAdminClient(aliasedURL)
 	fatalIf(err, "Cannot get a configured admin connection.")
 
-	// Fetch the server info of the specified Minio server
-	serverInfo, e := client.ServerInfo()
+	// Fetch info of all servers (cluster or single server)
+	serversInfo, e := client.ServerInfo()
```

Comment:
what happens when `serverOffline` is true, does the Addr field empty is handled as well?